examples/tabs.rs

```diff
@@ -5,31 +5,72 @@ use crossterm::{
     execute,
     terminal::{disable_raw_mode, enable_raw_mode, EnterAlternateScreen, LeaveAlternateScreen},
 };
-use ratatui::{prelude::*, widgets::*};
+use ratatui::{prelude::*, style::palette::tailwind, widgets::*};
+use strum::{Display, EnumIter, FromRepr, IntoEnumIterator};
 
-struct App<'a> {
-    pub titles: Vec<&'a str>,
-    pub index: usize,
+const TABS_COUNT: usize = 4;
+
+const PALETTES: [tailwind::Palette; TABS_COUNT] = [
```

Comment:
Can be simplified as:

```suggestion
const PALETTES: &[tailwind::Palette] = &[
```